core-services/image-mirroring/openshift/mapping_origin_4_7

```diff
@@ -199,6 +199,7 @@ registry.svc.ci.openshift.org/ocp/4.7:service-ca-operator quay.io/openshift/orig
 registry.svc.ci.openshift.org/ocp/4.7:service-catalog quay.io/openshift/origin-service-catalog:4.7 quay.io/openshift/origin-service-catalog:4.7.0
 registry.svc.ci.openshift.org/ocp/4.7:special-resource-operator quay.io/openshift/origin-special-resource-operator:4.7 quay.io/openshift/origin-special-resource-operator:4.7.0
 registry.svc.ci.openshift.org/ocp/4.7:sriov-cni quay.io/openshift/origin-sriov-cni:4.7 quay.io/openshift/origin-sriov-cni:4.7.0
+registry.svc.ci.openshift.org/ocp/4.7:sriov-infiniband-cni quay.io/openshift/origin-sriov-infiniband-cni:4.7 quay.io/openshift/origin-sriov-infiniband-cni:4.7.0
```

Comment:
this file is sorted, so this line should be after sriov-dp-admission-controller

Reply:
fixed for both 4_7 and 4_6